flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/OracleSourceTest.java

```diff
@@ -226,6 +226,7 @@ public void go() throws Exception {
             assertFalse(state.contains("server_id"));
             assertFalse(state.contains("event"));
 
+            source.cancel();
```

Comment:
We could add a new method to help to close `DebeziumSourceFunction` right.

Reply:
I have noticed that source.close contains source.cancel, so I remove all the source.cancel before source.close